packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -1942,10 +1915,10 @@ void main() {
     expect(renderEditable.text.style.decoration, isNull);
   });
 
+  const enum HandlePosition {
```

Comment:
I think `const` can not be used before type declarations?

Reply:
Whoops, good catch.